virt/vm_networking/virt-connecting-vm-to-sriov.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="virt-connecting-vm-to-sriov"]
+= Connecting a virtual machine to an SR-IOV network
+include::_attributes/common-attributes.adoc[]
+:context: virt-connecting-vm-to-sriov
+
+toc::[]
+
+You can connect a virtual machine (VM) to a Single Root I/O Virtualization (SR-IOV) network by performing the following steps:
+
+. xref:../../virt/vm_networking/virt-connecting-vm-to-sriov.adoc#nw-sriov-configuring-device_virt-connecting-vm-to-sriov[Configure an SR-IOV network device.]
+. xref:../../virt/vm_networking/virt-connecting-vm-to-sriov.adoc#nw-sriov-network-attachment_virt-connecting-vm-to-sriov[Configure an SR-IOV network.]
+. xref:../../virt/vm_networking/virt-connecting-vm-to-sriov.adoc#virt-attaching-vm-to-sriov-network_virt-connecting-vm-to-sriov[Connect the VM to the SR-IOV network.]
+
+[id="connecting-vm-to-sriov-network_virt-connecting-vm-to-sriov"]
+== Connecting a virtual machine to an SR-IOV network
+
+include::modules/nw-sriov-configuring-device.adoc[leveloffset=+2]
+
+include::modules/nw-sriov-network-attachment.adoc[leveloffset=+2]
+
+include::modules/virt-attaching-vm-to-sriov-network.adoc[leveloffset=+2]
+
+[id="next-steps_virt-connecting-vm-to-sriov"]
+== Next steps
+
+* xref:../../virt/vm_networking/virt-configuring-dpdk-workloads.adoc#virt-configuring-dpdk-workloads[Configuring DPDK workloads for improved performance]
```

Comment:
I would mention it as an additional resource. Even though SR-IOV is used in our DPDK config, most SR-IOV users would not use DPDK.